themes/gatsby-theme-blog/src/theme/prism.js

```diff
@@ -0,0 +1,48 @@
+// todo: swap for theme colors
```

Comment:
Is this still a TODO?

Reply:
It could be considered done for now. The prism colors are defined in `theme.colors.prism`, but I was originally thinking that maybe these colors could come from the base color palette instead of being defined separately